### PR TITLE
Adds bash adapter

### PR DESCRIPTION
**Pull request by aidenwallis**

### CODE REVIEW

**Comment by tolekk** [comment]
Code ends abruptly in line 3, I would suggest adding a longer piece of code based on the BEE source used for this.

**Review by styler — APPROVED** [review]
Thanks!

File: adapters/ba.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```
Comment (RAnders00) [inline adapters/ba.sh:1]:
```suggestion
#!/home/randers/Downloads/bash-0.3.2/out/bin/bash
```

Comment (aidenwallis) [reply adapters/ba.sh:1]: no

Comment (RAnders00) [reply adapters/ba.sh:1]: I will report you to the hacktoberfest police 😡

**Pull request merged**